examples/zkapps/04-zkapp-browser-ui/contracts/src/Add.test.ts

```diff
@@ -57,19 +63,20 @@ describe('Add', () => {
 
   it('generates and deploys the `Add` smart contract', async () => {
     const zkAppInstance = new Add(zkAppAddress);
-    await localDeploy(zkAppInstance, zkAppPrivateKey, deployerAccount);
+    await localDeploy(zkAppInstance, zkAppPrivateKey, deployerAccount, deployerKey);
     const num = zkAppInstance.num.get();
-    expect(num).toEqual(Field.one);
+    expect(num).toEqual(Field(1));
   });
 
   it('correctly updates the num state on the `Add` smart contract', async () => {
     const zkAppInstance = new Add(zkAppAddress);
-    await localDeploy(zkAppInstance, zkAppPrivateKey, deployerAccount);
-    const txn = await Mina.transaction(deployerAccount, () => {
+    await Add.compile();
```

Comment:
if we turned off proving as suggested above, we can also remove `compile()` here (to make the tests run faster)